marketplace/deployer_helm_tiller_base/test/chartmuseum-marketplace/requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+- name: marketplace-k8s-app-tools-helm-hooks
+  version: 0.0.2
+  repository: gs://cloud-marketplace-tools-charts/
+- name: chartmuseum
```

Comment:
wait, we don't include a copy of this. Does that mean that `helm install`--i.e. the deployer--downloads it at runtime?